app/assets/stylesheets/_state-file.scss

```diff
@@ -516,7 +516,7 @@
     }
   }
 
-  .reveal__link {
+  .reveal__button {
```

Comment:
I still see some instances of `reveal__link` in the code base, should we change these? @arinchoi03 
<img width="1491" alt="Screenshot 2024-10-28 at 4 14 49 PM" src="https://github.com/user-attachments/assets/0bb3f8c2-eb18-4bd1-a73c-b445e03d6ca4">

Reply:
ooh these must be new ones that got merged in from `main`. I'll fix!